### PR TITLE
Added ShortList sniff

### PR DESCRIPTION
**Pull request by Majkl578**
`list(...)` -> `[...]`.

cc @carusogabriel

### CODE REVIEW

File: tests/Sniffs/PHP/data/shortListErrors.fixed.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		$b,
+		[$c],
+		$d,
+	],
+] = foo();
+[
+	'a' => [
+		'b' => [$b],
+	],
+] = foo();
```
Comment (Majkl578) [inline tests/Sniffs/PHP/data/shortListErrors.fixed.php]: PhpStorm is confused by the block above, but it is valid: https://3v4l.org/dAEfH.

**Comment by kukulich** [comment]
Perfect, this was on my todo list too :)

**Comment by carusogabriel** [comment]
@majkl578 I’ll open a WIP in doctrine’s CS with this one :)

**Comment by Majkl578** [comment]
> this was on my todo list

Mind to share that list, i.e. separate issue per each new sniff (or one issue with tasks)? :blush:

**Pull request merged**